android/src/main/java/com/airbnb/android/react/maps/AirMapView.java

```diff
@@ -578,13 +578,16 @@ public boolean dispatchTouchEvent(MotionEvent ev) {
 
         switch (action) {
             case (MotionEvent.ACTION_DOWN):
-                this.getParent().requestDisallowInterceptTouchEvent(true);
+                if (map != null && map.getUiSettings().isScrollGesturesEnabled()) {
+                    this.getParent().requestDisallowInterceptTouchEvent(true);
```

Comment:
Oh wait, I may have misread your message. I'm going to build and run Static Map now.